doc/api.md

```diff
@@ -162,22 +162,22 @@ Returns the canonical user object for the user whose token was used to make the
 **UNIMPLEMENTED**
 
 `id` is a base-10 integer corresponding to the `db_id` of the user you wish to access.
-Requires an authentication token granting membership privileges to access any user that is not the user whose token was used to make the request.
+Requires an authentication token granting view permissions for the user.
```

Comment:
Oh, yeah, so we don't have a `users.view` cap that is separate from `users.list`. `users.list` presumably allows you to see the details for all listed users, so we need something more fine-grained. I'd propose

```prolog
cap(Id, users.view(Id)).
cap(Id, users.view(_)) :- cap(Id, users.list).
```

Reply:
Ugh, really we should have `users.view(Id, Property)`; e.g. the discord bot shouldn't have access to people's card numbers etc.

And _really_ that should apply to searches too, but I can't think of a way to do that sanely...

If we aren't doing fine-grained permissions for search, it'd ideally be separate from view so that could do fine-grained permissions.

If that seems OTT though, I'm fine with yours.